data/alpha/field/official-name.yaml

```diff
@@ -3,4 +3,4 @@ datatype: string
 field: official-name
 phase: alpha
 register: ''
-text: Official name of country
+text: Official name of an entry
```

Comment:
It was copied from http://field.discovery.openregister.org/record/official-name. Maybe @psd could correct this. I just didn't want it referencing country since multiple registers in alpha have an official-name.